File: Java/Easy/1/1.java
class Solution {
    public int[] twoSum(int[] nums, int target) {

        for (int i = 0; i < nums.length; i++) {
            int x = target - nums[i];
            for (int j = i + 1; j < nums.length; j++) {
                if (nums[j] == x) {
                    return new int[] { i, j };
                }
            }
        }

        return null;
    }
}

File: Go/Easy/1/1.go
func twoSum(nums []int, target int) []int {
    solution := make(map[int] int)

    for i, num := range nums {
        var x = target - nums[i]
         if index, found := solution[x]; found {
            return []int{index, i}
        }
        solution[num] = i
    }


    return nil
}
